scripts/scripts/japanese_to_romaji.py

```
@@ -9,26 +9,29 @@
 import json
 import threading
+from string import punctuation
 from concurrent.futures import ThreadPoolExecutor, as_completed
 from pathlib import Path
-from typing import Dict, List, Any, Tuple
+from typing import Dict, List, Any, Tuple, Set
 from pydantic import BaseModel
 
 from library.gemini_client import get_global_gemini_client
 
 SYSTEM_PROMPT = """Convert the following Japanese text to Romaji using the Hepburn romanization system.
```

Pylint (3.3.7)

[convention] Line too long (103/100)

(C0301)